dolphin/api/v1/storages.py

```diff
@@ -127,6 +132,27 @@ def sync_all(self, req):
         return dict(name="Sync all storages")
 
     def sync(self, req, id):
+        """
+        :param req:
+        :param id:
+        :return:
+        """
+        # validate the id
+        context = req.environ.get('dolphin.context')
+        # admin_context = context.RequestContext('admin', 'fake', True)
+        try:
+            device = db.access_info_get(context, id)
+        except exception.NotFound as e:
+            LOG.error(e)
+            raise exception.DolphinException(e)
```

Comment:
raise e directly here, NotFound is subclass of DolphinException.

Reply:
done